src/build/app/mod.rs

```diff
@@ -1272,7 +1272,7 @@ impl<'b> App<'b> {
                 }
                 drop(self);
                 drop(e);
-                process::exit(exitcode::USAGE);
+                process::exit(1);
```

Comment:
```suggestion
                process::exit(64);
```

Reply:
What the exit code should be is a good question. Let's take a look at other unix apps first and take the most common option.

I really don't think 64 is of any value here.

Reply:
64 is the `USAGE` exit code which has been proposed in #1327

Reply:
I can't really agree with this proposal because I have yet to see an app that exits with 64 on usage.

Some linux apps (just tested)

app | invalid | `--help`
---  | ---  | ---  | 
`ls` | 2 | 0 
`cargo` | 1 | 0
`cat` |1|0
`xargs` |1|0
`grep`|2|0
`awk`|1|0
`curl`|2|0
`wget`|2|0

I realize this table is not representative, I just wanted to show that apps do not respect `sysexits.h` at all.

Thus, I propose using 2 since I agree it should be different from other errors.